sunpy/coordinates/__init__.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import absolute_import
```

Comment:
Just out of curiosity, any reason to not include ``division``? Also, some files below are missing the future import, I think you should always include it (especially division)